packages/components/CHANGELOG.md

```diff
@@ -15,6 +15,7 @@
 ### Internal
 
 -   `Flex`, `FlexBlock`, and `FlexItem` components have been re-written from the ground up ([#31297](https://github.com/WordPress/gutenberg/pull/31297)).
+-   `color()` utility function has been removed, in favour of accessing color values directly ([#31661](https://github.com/WordPress/gutenberg/pull/31661))
```

Comment:
Nit: we're mixing American and British English when using `color` (US) but `favour` (UK). I guess we should stick to the American version as it's the one used in web terms.

```suggestion
-   `color()` utility function has been removed, in favor of accessing color values directly ([#31661](https://github.com/WordPress/gutenberg/pull/31661))
```

Reply:
Well spotted! I normally write in British English, and so this happens more often than I'd like 😅

Actually, in light of @diegohaz 's [comment on a very similar PR](https://github.com/WordPress/gutenberg/pull/31646#discussion_r630395238), should I actually just delete this changelog entry?

Reply:
Yup, agree that changes in non-exposed internals don't need to be documented here 👍

Reply:
Done in https://github.com/WordPress/gutenberg/pull/31661/commits/dd4b976853827c3311f802232d9800f7103b7870